.travis.yml

```diff
@@ -97,12 +97,12 @@ matrix:
           sudo: required
           compiler: gcc
           node_js:
-            - "7"
+            - "8"
```

Comment:
Not sure we should use this plus nvm install. One or the other should be enough.